src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-top: 0;
+  margin-right: 4px;
+  justify-content: center;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Even though you haven't set a `font-family` here, ensure that if you do, you include a generic family as a fallback.